core/contracts/financial-templates/implementation/PricelessPositionManager.sol

```diff
@@ -149,14 +149,14 @@ contract PricelessPositionManager is FeePayer, AdministrateeInterface {
         FixedPoint.Unsigned memory _minSponsorTokens,
         address _timerAddress
     ) public FeePayer(_collateralAddress, _finderAddress, _timerAddress) {
+        require(_expirationTimestamp > getCurrentTime());
```

Comment:
This PR seems to conflict with #1305 since that adds require messages to old requires and this one introduces new requires without messages. Could you and @chrismaree decide whether to go ahead and add messages to the new requires in this PR or to have #1305 merge after this PR and add requires for these too?

Reply:
I talked with Chris about this and we decided to keep #1305 frozen until we get all EMP audit issues finished so that we can fix bytecode issues in one PR.

Reply:
SGTM.